### ISSUE TITLE
Debug Metrics

### ISSUE THREAD

**Issue by SyntaxNode** [prebid/prebid-server#1474]
### Motivation
This proposal addresses a discussion in https://github.com/prebid/prebid-server/issues/745 to provide hosts with visibility in how many auctions are running with debugging enabled.

### Feature Request
Add new general metrics and new account metrics to track the number of requests received with debugging enabled through either the test or debug flag.

1. Add a new counter metric `requests_debug` which increments whenever a new request with debugging enabled is received. May be compared with the existing `requests` counter metric to determine percentages.

2. Add a new account metric `account_requests_debug` which increments whenever a new request with debugging enabled for a specific account is received. May be compared with the existing `account_requests` counter metric to determine percentages. Since account cardinality may be very high, this should be controlled with an option.

3. Add a new `metrics.disabled.account_debug` config option to control enabling the account metric. Due to potentially very high account cardinality, this option will default to `true`, matching the default for `metrics.disabled.account_adapter_details`.

**Comment by SyntaxNode** [comment]
Yes, this was completed for PBS-Go in https://github.com/prebid/prebid-server/pull/2246.

**Comment by bretg** [comment]
Confirmed this is not yet ported to PBS-Java

**Comment by bsardo** [comment]
@osulzhenko please classify priority.
